src/cf-pagination/src/molecules/pagination.less

```diff
@@ -0,0 +1,107 @@
+//
+// Theme variables
+//
+
+// Color variables
+// $color- variables are from 18F's US Web Design Standards
+// https://github.com/18F/web-design-standards/blob/18f-pages-staging/src/stylesheets/core/_variables.scss
+
+@pagination-text-color: #757575; // $color-gray-medium
+@pagination-bg-color: #f1f1f1; // $color-gray-lightest
+
+// Sizing variables
+
+@pagination-font-size__px: @base-font-size-px;
+@pagination-font-size__em: unit( @pagination-font-size__px / @base-font-size-px, em );
+@pagination-height__px: 42px;
+@pagination-width__px: 130px;
+
+//
+// Pagination
+//
+
+.m-pagination {
+    position: relative;
+
+    &_form {
```

Comment:
Trivial: We should probably not call this `_form` as the appearance would still be the same whether it was applied to `<form>` or ` <div>`, right? The CSS shouldn't really care whether it's a form or not, it's just setting some colors and padding on the container around the buttons.